dashboard/app/views/pd/workshop_mailer/_csd_pre_course_reading.html.haml

```diff
@@ -0,0 +1,28 @@
+%h4
```

Comment:
I was hoping to make these two one file, but they are just too different on link content. Two files seemed cleaner